src/main/resources/templates/room.html

```diff
@@ -108,13 +110,48 @@
         </ul>
         </div>
 
+        <!--<button type="button" class="btn btn-outline-secondary" data-bs-toggle="modal" data-bs-target="#goalModal">🎯</button>-->
+        <div class="btn-group">
+          <button class="btn btn-outline-secondary" type="button" data-bs-toggle="modal" data-bs-target="#goalModal">🎯</button>
+          <button type="button"
+                  class="btn btn-outline-secondary dropdown-toggle dropdown-toggle-split"
+                  data-bs-toggle="dropdown"
+                  aria-expanded="false">
+            <span class="visually-hidden">Toggle Dropdown</span>
+          </button>
+          <ul class="dropdown-menu">
+            <li><a class="dropdown-item" onclick="deleteGoal()">🗑️ Delete current goal</a></li>
```

Comment:
I would name this "Clear" and basically clear the textbox, which would trigger a change event sending an empty string as a goal. Keep it simple.